src/server.tsx

```diff
@@ -104,4 +102,22 @@ const startServer = async () => {
   });
 };
 
+function getOpenRobotsTxt() {
+  const robotstxt = ['User-agent: *'];
+  const AI = GameMode.AI;
+  const OnlineFriend = GameMode.OnlineFriend;
+  GAMES_LIST.forEach((game) => {
+    game.modes.forEach((modeInfo) => {
+      if (modeInfo.mode === GameMode.AI) {
+        robotstxt.push(`Disallow: /g/${game.code}/ai/`);
```

Comment:
isnt AI supposed to be capital letters here?

Reply:
Yes, you're right -- and robots.txt **is** case sensitive!  Thank you for catching!